### DISCUSSION TITLE
Enable PKCE option for openid

### DISCUSSION THREAD

**Discussion by D3an1el** [hargata/lubelog#568] [Ideas]
Hi, is it possible to add PKCE option for openid as an option as Kanidm doesnt allow OIDC without PKCE for security reassons?.

- https://auth0.com/docs/get-started/authentication-and-authorization-flow/authorization-code-flow-with-pkce

- https://kanidm.github.io/kanidm/master/frequently_asked_questions.html#oauth2

i did find this https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authentication.oauth.oauthoptions.usepkce?view=aspnetcore-8.0 but not sure where or how too  add it in lubelog.

**Comment by hargata** [comment]
Done, released in 1.3.6

Add the following environment variable and LubeLogger will generate and authenticate a PKCE key pair
`OpenIDConfig__UsePKCE=true`
